Wikipedia/Code/WMFColumnarCollectionViewLayout.m

```diff
@@ -103,6 +103,14 @@ - (void)prepareLayout {
     [super prepareLayout];
 }
 
+- (CGPoint)targetContentOffsetForProposedContentOffset:(CGPoint)proposedContentOffset {
```

Comment:
Do these methods need to be implemented if you aren't modifying the content offset?